plugin/src/sidebar.ts

```diff
@@ -253,6 +253,9 @@ reearth.on("message", ({ action, payload }: PostMessageProps) => {
   } else if (action === "modalClose") {
     reearth.modal.close();
     welcomePageIsOpen = false;
+  } else if (action === "openDatasetDetails") {
+    reearth.modal.show(dataCatalogHtml, { background: "transparent" });
+    reearth.modal.postMessage({ action: "openDatasetDetails", payload });
```

Comment:
Sorry can you use the catalogModalOpen action, and just do a check 
```
if(payload){
   reearth.modal.postMessage({ action: "openDatasetDetails", payload });
}
```

Reply:
We already have so many actions. Don't want to add more unless completely necessary..

Reply:
Done!